portal/code/cx-portal/src/components/shared/frame/NavMenu/index.tsx

```diff
@@ -5,13 +5,19 @@ import './NavMenu.scss'
 export const NavMenu = ({
   pages,
   horizontal,
+  plain,
 }: {
   pages: string[]
   horizontal?: boolean
+  plain?: boolean
 }) => {
   const { t } = useTranslation()
   return (
-    <nav className={`NavMenu ${horizontal ? 'horizontal' : 'vertical'}`}>
+    <nav
+      className={`NavMenu ${horizontal ? 'horizontal' : 'vertical'} ${
+        plain ? 'plain' : 'fancy'
```

Comment:
It is hard to understand what is the difference between 'plain' and 'fancy'. Maybe there are more descriptive keywords what it affects.